index.bs

```diff
@@ -10015,35 +10135,60 @@ interface AudioWorkletProcessor {
 
 <h5 id="AudioWorketProcessor-constructors">
 Constructors</h5>
+
 <dl dfn-type="constructor" dfn-for="AudioWorkletProcessor">
 	: <dfn>AudioWorkletProcessor(options)</dfn>
 	::
-		When the constructor for {{AudioWorkletProcessor}} is invoked, the following steps are performed:
+		When the constructor for {{AudioWorkletProcessor}} is invoked,
+		the following steps are performed on the <a>rendering thread</a>.
```

Comment:
Is this mandatory ? It would be best if this was not done on an RT thread, right?

Reply:
Hmm. AudioWorkletProcessor belongs to the rendering thread. How would you use this object if this is created in somewhere else? Are you creating this from the control and transfer it?

Reply:
If you want to have the flexibility of "construction", you might want to create a follow-up PR after this one.